tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "lib": ["es2022"],
     "rootDir": "src",
     "outDir": "dist",
-    "typeRoots": ["mocha", "node", "node_modules/@types", "@types"],
+    "typeRoots": ["node", "node_modules/@types"],
```

Comment:
shouldn't we include
```suggestion
    "typeRoots": ["node", "node_modules/@types", "..node_modules/@types"],
```
?

Reply:
Ah no this is still a flat project so no need thanks